.github/workflows/tests.yaml

```diff
@@ -35,6 +35,10 @@ jobs:
       run: |
         source miniforge3/bin/activate ceesd
         ./version.sh ./mirgecom/requirements.txt
+        
+        # Don't build pyopencl as that is difficult
+        cat requirements.txt.* | sed /pyopencl/d > req.txt
```

Comment:
Since we install it via conda, why is it in `requirements.txt` in the first place? (I ask because likely any user will run into issues building it.)

Reply:
I think because not everyone will use emirge or conda to install mirgecom?

Reply:
Fair enough. Though maybe document (in the other PR) that it can be a stumbling block and can, in most cases, be excluded from the build.

Reply:
Done.